package.json

```diff
@@ -192,9 +193,10 @@
         "@web3-onboard/core@^2.8.5": "patch:@web3-onboard/core@npm%3A2.8.5#./.yarn/patches/@web3-onboard-core-npm-2.8.5-10c73a002c.patch"
     },
     "dependencies": {
+        "@gooddollar/good-design": "file:.yalc/@gooddollar/good-design",
         "@gooddollar/goodprotocol": "^1.0.29-beta.4",
-        "@gooddollar/web3sdk": "^0.1.11",
-        "@gooddollar/web3sdk-v2": "^0.1.29",
+        "@gooddollar/web3sdk": "file:.yalc/@gooddollar/web3sdk",
```

Comment:
this look like local running/debugging stuff right ? 
please make sure to adjust it before merging PR

Reply:
yes, first 242 needs to be merged in sdkv2